components/login/videoCard.tsx

```diff
@@ -6,66 +6,60 @@ import {
   Text,
   useColorModeValue,
   Image,
+  Link,
 } from "@chakra-ui/react";
+// import Link from "next/link";
 import React from "react";
 
+export interface IVideoCard {
+  imageUrl: string;
+  imageAlt: string;
+  title: string;
+  channel: string;
+  id: string;
+}
+
 export default function VideoCard({
   imageUrl,
   imageAlt,
-  date,
-  views,
   title,
-}: any) {
+  channel,
+  id,
+}: IVideoCard) {
   return (
     <Flex justifyContent={"center"}>
-      <Box bg={useColorModeValue("white", "gray.800")} maxW="sm">
-        <Image src={imageUrl} alt={imageAlt} rounded="lg" />
-        <Box py="2">
-          <Box d="flex" alignItems="center">
-            <Avatar
-              my={1}
-              size="sm"
-              name="anubra266"
-              src="https://avatars.githubusercontent.com/u/68416000?v=4"
-              cursor="pointer"
-            />
+      <Link href={`/video/${id}`}>
+        <Box bg={useColorModeValue("white", "gray.800")} maxW="sm">
+          <Image src={imageUrl} alt={imageAlt} rounded="lg" />
+          <Box py="2">
+            <Box d="flex" alignItems="center">
+              <Avatar my={1} size="sm" name={channel} src="" cursor="pointer" />
 
+              <Box
+                color={useColorModeValue("gray.800", "#F5F8FA")}
```

Comment:
Use a global theme and then make use of colors i.e `gray.500` or something else, Avoid usage of hex code values

Reply:
Extended the Chakra theme, so wouldn't be using hash colors anywhere.